.travis.yml

```diff
@@ -3,7 +3,8 @@ python:
   - "2.7"
   - "3.5"
   - "3.6"
-  - "pypy-5.4"
+  - "3.7"
+  - "pypy3.6-7.0.0"
```

Comment:
The previous python versions seem to be a little outdated. I guess that's why you'd like to update the versions. 
May we move the change to a separate PR? This PR concentrates on the OAuth-related mechanism instead of python versions.

Reply:
The old version breaks tests.

Reply:
I see. Thanks.